docs/samples/drift-detection/alibi-detect/cifar10/cifar10_drift.ipynb

```diff
@@ -383,33 +387,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "tfserving-cifar10.cifar10.example.com\n"
+      "tfserving-cifar10-predictor-kserve-demo.apps.aliok-c095.serverless.devcluster.openshift.com\n"
      ]
     }
    ],
    "source": [
-    "SERVICE_HOSTNAMES=!(kubectl get inferenceservice -n cifar10 tfserving-cifar10 -o jsonpath='{.status.url}' | cut -d \"/\" -f 3)\n",
+    "SERVICE_HOSTNAMES=!(kubectl get ksvc -n cifar10 tfserving-cifar10-predictor -o jsonpath='{.status.url}' | cut -d \"/\" -f 3)\n",
```

Comment:
I think we’d still want to get the url from isvc, I know there was a bug when top level virtual service is turned off but it should be fixed now.